internal/cluster/validator.go

```diff
@@ -211,6 +235,9 @@ func (v *clusterValidator) printIsApiVipValid(context *clusterPreprocessContext,
 }
 
 func (v *clusterValidator) isIngressVipDefined(c *clusterPreprocessContext) validationStatus {
+	if swag.BoolValue(c.cluster.UserManagedNetworking) {
```

Comment:
We should find a way to remove variations if they are irrelevant

Reply:
I followed day2 logic:
https://github.com/openshift/assisted-service/blob/master/internal/host/validator.go#L451

Having a mechanism to update the state machine based on a parameter is out of scope here.
It should be part of handling "Profiles" epic.

WDYT?

Reply:
@ori-amizur i'm not sure that i understand what you mean, if we remove validation how do you expect that state machine conditions table to look like?

Reply:
@filanov   What I meant that if we leave it the way it is, in couple of months we will have many validations that are irrelevant.   Maybe it is not so bad because the UI doesn't show successful validations, but it is a workaround.

Reply:
It is probably not part of this PR

Reply:
I understand what you mean, my question is how did you imagine the solution?

Reply:
Don't know - something to think about